org.moneymanagerex.MMEX.yml

```
@@ -0,0 +1,42 @@
+app-id: org.moneymanagerex.MMEX
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: mmex
+rename-desktop-file: mmex.desktop
+rename-icon: mmex
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=home
+  - --share=ipc
+modules:
+  - name: wxWidgets
+    config-opts:
+       - --disable-shared
+       - --enable-cxx11
+       - --with-cxx=11
+       - --enable-webview
+       - --with-gtk=3
+       - --enable-unicode
+       - --enable-intl
+    sources:
+       - type: archive
+         url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.2.2.1/wxWidgets-3.2.2.1.tar.bz2
+         sha256: dffcb6be71296fff4b7f8840eb1b510178f57aa2eb236b20da41182009242c02
+  - name: moneymanagerex
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    post-install:
+      - desktop-file-edit --set-icon="${FLATPAK_ID}" ${FLATPAK_DEST}/share/applications/mmex.desktop
```

```suggestion
    post-install:
      - desktop-file-edit --set-icon="${FLATPAK_ID}" ${FLATPAK_DEST}/share/applications/mmex.desktop
```

`rename-icon` takes care of this.